src/azul/service/manifest_service.py

```diff
@@ -639,6 +639,7 @@ def _option(self, s: str):
     def write_to(self, output: IO[str]) -> Optional[str]:
         output.write('--create-dirs\n\n')
         output.write('--compressed\n\n')
+        output.write('--location\n\n')
```

Comment:
```suggestion
        output.write('--create-dirs\n\n'
       '--compressed\n\n')
       '--location\n\n')
```

with proper indentation